raiden/transfer/mediated_transfer/initiator.py

```diff
@@ -721,6 +722,40 @@ def handle_onchain_secretreveal(
     return iteration
 
 
+def handle_onchain_secretreveal_light(
```

Comment:
comparing this function to `handle_onchain_secretreveal`, 2 things:
1. could you briefly explain why this function does not care about events? (so that i can maybe understand the logic here a bit better)
2. should the docstrings be the same if the functions are different?

Reply:
1. It just don't send the balance proof, as it would have been needed to modify the light client, and that would be too much time expensive without debray.

Reply:
1. It just don't send the balance proof, as it would have been needed to modify the light client, and that would be too much time expensive without debray.
2. No, they shouldn't. I'll update them

Reply:
1. basically it only update the local offchain state of the channel in order to allow the unlock to be fired right?

but the initiator (the lc) didnt handle this, therefore the balance proof isnt sent to the partner

Reply:
exactly what marcos says